openhtf/io/output/console_summary.py

```diff
@@ -0,0 +1,107 @@
+"""Module to display test summary on console."""
+
+import os
+import sys
+import __main__ as main
```

Comment:
Good point. This line was my suggestion, but `station_id` is at least as good here and possibly better. I'm cool either way.